code-compass.el

```diff
@@ -328,280 +331,287 @@
 
-(defun c/run-code-maat (command repository)
+(defun code-compass--run-code-maat (command repository)
   "Run code-maat's COMMAND on REPOSITORY."
   (message "Producing code-maat %s report for %s..." command repository)
-  (let ((source-file (format "%s/code-maat-1.0.1-standalone.jar" c/download-directory))
-        (maat-jar-p (s-contains-p "jar" c/code-maat-command)))
-    (when (and maat-jar-p (not (file-exists-p (c/expand-file-name source-file))))
-      (mkdir c/download-directory t)
-      (url-copy-file "https://github.com/smontanari/code-forensics/raw/v3.0.0/lib/analysers/code_maat/code-maat-1.0.1-standalone.jar" (c/expand-file-name source-file) t))
+  (let ((source-file (format "%s/code-maat-1.0.1-standalone.jar" code-compass-download-directory))
+        (maat-jar-p (s-contains-p "jar" code-compass-code-maat-command)))
+    (when (and maat-jar-p (not (file-exists-p (code-compass--expand-file-name source-file))))
+      (mkdir code-compass-download-directory t)
+      (url-copy-file "https://github.com/smontanari/code-forensics/raw/v3.0.0/lib/analysers/code_maat/code-maat-1.0.1-standalone.jar" (code-compass--expand-file-name source-file) t))
     (shell-command
      (format
       "%1$s -l %4$s/code-compass-%2$s/gitreport.log -c git2 -a %3$s > %3$s.csv"
-      c/code-maat-command
+      code-compass-code-maat-command
       (f-filename repository)
       command
-      (if maat-jar-p c/tmp-directory c/docker-data-directory))
+      (if maat-jar-p code-compass-tmp-directory code-compass-docker-data-directory))
      nil
-     (get-buffer-create "*c/run-code-maat-errors*")
+     (get-buffer-create "*code-compass--run-code-maat-errors*")
      )
     (if-let* ((contents
-               (with-current-buffer "*c/run-code-maat-errors*"
+               (with-current-buffer "*code-compass--run-code-maat-errors*"
                  (buffer-string)))
               (_ (> (length contents) 1)))
-        (error (concat "c/run-code-maat\n\n" contents)))))
+        (error (concat "code-compass--run-code-maat\n\n" contents)))))
 
-(defun c/produce-code-maat-revisions-report (repository)
+(defun code-compass--produce-code-maat-revisions-report (repository)
   "Create code-maat revisions report for REPOSITORY."
-  (c/run-code-maat "revisions" repository)
+  (code-compass--run-code-maat "revisions" repository)
   repository)
 
-(defun c/produce-cloc-report (repository)
-  "Create cloc report for REPOSITORY. To filter specific subdirectories out of this report, edit the variable `c/exclude-directories'."
+(defun code-compass--produce-cloc-report (repository)
+  "Create cloc report for REPOSITORY. To filter specific subdirectories out of this report, edit the variable `code-compass-exclude-directories'."
   (message (concat
             "Producing cloc report with "
             (format "(cd %s; PERL_BADLANG=0 cloc ./ --by-file --csv --quiet --exclude-dir=%s) > cloc.csv" repository (string-join c/exclude-directories ","))
```

Comment:
could you fix this with `code-compass-exclude-directories`?

Reply:
done